ext/php8/ddtrace_config.c

```diff
@@ -0,0 +1,56 @@
+#include "ddtrace_config.h"
+
+#include <assert.h>
+#include <ctype.h>
+#include <stdio.h>
+#include <string.h>
+
+#define DD_CONFIG_ENTRY(name, type, default) ZAI_CONFIG_ENTRY(DDTRACE_CONFIG_##name, name, type, default)
+#define DD_CONFIG_ALIASED_ENTRY(name, type, default, aliases) \
+    ZAI_CONFIG_ALIASED_ENTRY(DDTRACE_CONFIG_##name, name, type, default, aliases)
+
+static void dd_ini_env_to_ini_name(zai_string_view env_name, zai_config_ini_name *ini_name) {
```

Comment:
When I read this function I just think ... does it really need 30 LoC to do `strtolower(strtr($input, ["DD_TRACE_" => "datadog.trace", "DD_" => "datadog."]))` in C? :-D Maybe it does...

Reply:
This is the story of my life. 😆